src/views/Entries/QualityControl/index.tsx

```diff
@@ -29,7 +37,22 @@ function QualityControl(props: QualityControlProps) {
                 rightContainerClassName={styles.right}
                 rightChild={(
                     <div className={styles.entryList}>
-                        Entry list
+                        { leadGroupedEntriesList.map((leadWithEntries) => {
```

Comment:
Use list components

Reply:
This is only temporary and will be changed later